packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -414,3 +414,33 @@ test.describe('`position`', () => {
         });
     });
 });
+
+test.describe('Prop: `hasStackedActions`', () => {
+    test.describe('when true', () => {
```

Comment:
Is there a test for false?

Reply:
I haven't made one, as after chatting it through with a couple of others, it seems like the visual tests for `false` would be covered by every other test being run (as that's the default, so the default modal test effectively tests the `false` state for instance). So didn't want to duplicate and add another set of tests that could be unecessary.

Reply:
We should probably be careful how many visual tests we add as well as it increases both CI duration and usage